### ISSUE TITLE
Firefox Cache Issue

### ISSUE THREAD

**Issue by JRDeLisle** [rlh-aagis/vacant2vibrant#51]
I just checked the Map Search and new .25 and like the zoom and property display. It worked in Chrome. In Firefox I had to clear my cache which most will not know how to do and/or that they need to when a feature doesn't work. It does work on Android without cache issue.  Also works in Explorer. Not sure if it's a browser issue or just because we're changing and will self-correct.

**Comment by andreiradulescu** [comment]
I have placed additional cache controls on the website, so now it should not cache as hard. This will need  testing on a number of different systems.

**Comment by JRDeLisle** [comment]
Thanks. I can try Chrome, Firefox, and Explorer on Windows as well as on Android. I do not have Apple products to test. Anyone else have access to them. I know my students do so let me know when to get them testing.

**Comment by andreiradulescu** [comment]
Yes, please go ahead and have your students test for caching problems now.

**Comment by JRDeLisle** [comment]
I'm still running into some with Chrome and Firefox; have to manually clear but then it works. Only issue is some may not see the new enhancements.
